src/test/pythonnode_test.cpp

```diff
@@ -518,7 +728,112 @@ TEST_F(PythonFlowTest, PythonNodeLoopbackDefinedOnlyOnOutput) {
     mediapipeDummy.inputConfig = testPbtxt;
     ASSERT_EQ(mediapipeDummy.validate(manager), StatusCode::MEDIAPIPE_GRAPH_INITIALIZATION_ERROR);
 }
-// TODO: Add test with only input LOOPBACK
+
+// MediaPipe does allow such connection and treats loopback as connected (due to the back edge)
+// However, such graph will hang since nothing feeds loopback with initial data
+// During the hang, the graph node will wait for secondary input (besides "input")
+TEST_F(PythonFlowTest, DISABLED_PythonNodeLoopback_SyncSet_Missing) {
```

Comment:
It works as designed. I wonder if we will ever be able to enable that test. If user configures graph against our rules for generative mode it's expected to happen. Do you think we should add our own validation there in the future?

Reply:
I think we should treat MP as black box and possibly introduce the fix in MP repo.